tests/test_trade_pipeline.py

```diff
@@ -1163,6 +1163,60 @@ def test_order_expires_after_only_maker_messaged(self):
             f"⚖️ Hey {data['taker_nick']}, a dispute has been opened on your order with ID {str(trade.order_id)}.",
         )
 
+    # def test_dispute_closed_maker_wins(self):
+    #     trade = Trade(self.client)
+    #     trade.publish_order()
+    #     trade.take_order()
+    #     trade.lock_taker_bond()
+    #     trade.lock_escrow(trade.taker_index)
+    #     trade.submit_payout_invoice(trade.maker_index)
+
+    #     trade.change_order_status(Order.Status.TLD)
+
+    #     trade.clean_orders()
+
+    #     maker_headers = trade.get_robot_auth(trade.maker_index)
+    #     response = self.client.get(reverse("notifications"), **maker_headers)
+    #     self.assertResponse(response)
+    #     notifications_data = list(response.json())
+    #     self.assertEqual(notifications_data[0]["order_id"], trade.order_id)
+    #     self.assertEqual(
+    #         notifications_data[0]["title"],
+    #         f"⚖️ Hey {data['maker_nick']}, you won the dispute on your order with ID {str(trade.order_id)}."
+    #     )
+    #     taker_headers = trade.get_robot_auth(trade.taker_index)
+    #     response = self.client.get(reverse("notifications"), **taker_headers)
+    #     self.assertResponse(response)
+    #     notifications_data = list(response.json())
+    #     self.assertEqual(notifications_data[0]["order_id"], trade.order_id)
+    #     self.assertEqual(
+    #         notifications_data[0]["title"],
+    #         f"⚖️ Hey {data['taker_nick']}, you lost the dispute on your order with ID {str(trade.order_id)}."
+    #     )
```

Comment:
Since the dispute resolution is an admin action, I'm not quite sure how to do it, ideas @Reckless-Satoshi ?

Reply:
I wonder whether we could directly call https://github.com/RoboSats/robosats/blob/dfd1cf53602bdcee14727661f27eea9bbd1f0a73/api/admin.py#L185 ... Looks difficult. Maybe we should explore how the request sent from the admin panel looks when solving a dispute and try to imitate it here?

Reply:
And fake the admin request with credentials? I'll take a look but for this kind of solutions they are always tricky